notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<p>Tutaj wydaje mi się, że można jeszcze połączyć ze sobą </p><p>a) <code>Fundacje</code>, <code>Stowarzyszenia</code>   i <code>Organizacje</code> </p><p>b) <code>Samorząd</code>,  <code>Miasto na prawach powiatu</code>,  <code>Powiat ziemski</code>,  <code>Gmina</code> </p><p>c) <code>Osoba_fizyczna</code>,  <code>Spółki Handlowe</code>,  <code>Przedsiębiorstwa/Izby/Spoldzielnie/Uczelnia</code></p><p> </p><p>Co do <code>Uczelnia</code>  to sam się&nbsp;zastanawiam czy to nie jest osobna kategoria skoro nie wiemy czy to jest uczelnia prywatna czy publiczna..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>